### PR TITLE
Fix typo in controlPlane JSON tag

### PR DESCRIPTION
**Pull request by munnerz**
Fix a typo in the Config structure's `controlPlane` field.

I'm not 100% sure how this effects existing users - investigating further...

### CODE REVIEW

**Comment by BenTheElder** [comment]
LOL, I guess the bot doesn't look at commits, only PRs /shrug

File: pkg/cluster/config/v1alpha1/types.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type Config struct {
 	// as patchesJson6902 to `kustomize build`
 	KubeadmConfigPatchesJSON6902 []kustomize.PatchJSON6902 `json:"kubeadmConfigPatchesJson6902,omitempty"`
 	// ControlPlane holds config for the control plane node
-	ControlPlane *ControlPlane `json:"ControlPlane,omitempty"`
+	ControlPlane *ControlPlane `json:"controlPlane,omitempty"`
```
[the remark this answers is not in the document]
Comment (munnerz) [reply pkg/cluster/config/v1alpha1/types.go]: Based on [this](https://github.com/kubernetes-sigs/kind/search?utf8=%E2%9C%93&q=ControlPlane&type=), it looks like you have removed it already but forgot the config field 😄 

Which also explains why my attempts to use it aren't working 🙃 I think this field just needs removing then.

[the remark this answers is not in the document]
Comment (BenTheElder) [reply pkg/cluster/config/v1alpha1/types.go]: yeah, if you have a good use case we can add it back, but my initial thinking is that we probably don't want to set the node lifecycle in stone this early, and it is part of the public API right now. maybe post v1?

**Comment by BenTheElder** [comment]
Let's go ahead and fix this anyhow 👍
/lgtm

**Pull request merged**